### PR TITLE
Shifting remote to plugins folder

### PR DESCRIPTION
**Pull request by RohanVashisht1234**

### CODE REVIEW

**Review by Guldoman — COMMENTED** [review]
This is missing the move for `language_swift` in the manifest.

**Comment by RohanVashisht1234** [comment]
> This is missing the move for `language_swift` in the manifest.

Done!

File: manifest.json [hunk 1]
```diff
@@ -1442,11 +1442,11 @@
       "description": "Syntax for the [Swift](https://developer.apple.com/swift/) programming language",
       "id": "language_swift",
       "mod_version": "3",
-      "remote": "https://github.com/RohanVashisht1234/swift_syntax_highlighting_lite-xl:4211079ced99ed693b5c8acad61f3a27a511919d",
+      "path":"plugins/language_swift",
```
Comment (Guldoman) [inline manifest.json:1445]:
```suggestion
      "path":"plugins/language_swift.lua",
```

**Comment by RohanVashisht1234** [comment]
ok, increased the bump version also

**Pull request merged**